update using core-store

sevenEng opened the pull request:
will have a test together with an updated app

sevenEng committed:
update using core-store

Toshbrown commented:
Thanks, @sevenEng this looks good, I will give it a test and merge it when we move to v0.3.0.

Toshbrown committed:
Update node-databox to 0.6.0-rc10

Fixes observe timeout

Toshbrown committed:
Working version 

Most tweets are rejected from the store due to UTF8 chars. https://github.com/jptmoore/zest/issues/15

Toshbrown committed:
Fix testActuator.DataSourceID

Toshbrown committed:
Update to node-databox 0.6.0-rc11

Fixes https://github.com/jptmoore/zest/issues/15

Toshbrown committed:
Reduce image size from 308MB to 93.1MB

Toshbrown committed:
Remove verbose debug output

Toshbrown committed:
Update version to 0.3.0

Toshbrown committed:
Update node-lib-databox version
